### PR TITLE
feat(advanced): add selection between flash/serialflash methods

### PR DESCRIPTION
**Pull request by CODeRUS**
Signed-off-by: Andrei Kozhevnikov <coderusinbox@gmail.com>

### CODE REVIEW

File: scripts/flash_klipper.sh [hunk 1]
```diff
@@ -52,16 +52,57 @@ function init_flash_process() {
...
+  top_border
+  echo -e "| How to flash MCU?                                     |"
+  echo -e "| 1) make flash (default)                               |"
+  echo -e "| 2) make serialflash (stm32flash)                      |"
+  blank_line
+  back_help_footer
+
+  local choice
+  while true; do
+    read -p "${cyan}###### Flashing command:${white} " choice
```
Comment (dw-0) [inline scripts/flash_klipper.sh:80]: Option "1" needs to be added as the default here, if we want to pre-fill the input with "1".

File: scripts/flash_klipper.sh [hunk 2]
```diff
@@ -52,16 +52,57 @@ function init_flash_process() {
...
+      2)
+        select_msg "Selected 'make serialflash' command"
+        flash_command="serialflash"
+        break;;
+      B|b)
+        advanced_menu
+        break;;
+      H|h)
+        clear && print_header
+        show_mcu_connection_help
```
Comment (dw-0) [inline scripts/flash_klipper.sh:95]: Im unsure why you choose to display the mcu_connection_help dialog here. I think we rather should show a new help-dialog which may explains the difference between the two options selectable in that new flash_command dialog as the mcu connection dialog comes in the next step?

File: scripts/flash_klipper.sh (hunk 1 above)
Comment (CODeRUS) [reply scripts/flash_klipper.sh]: Added default option

**Comment by dw-0** [comment]
Thank you!

**Pull request merged**